docs/index.rst

```diff
@@ -19,26 +17,26 @@ Key concepts
 
 **Primitives**
 
-Primitives are core functions that provide a simplified interface for defining `near-time quantum-classical workloads <https://research.ibm.com/blog/near-real-time-quantum-compute>`_  required to efficiently build and customize applications. The initial release of Qiskit Runtime includes two primitives: Estimator and Sampler. They perform foundational quantum computing tasks and act as an entry point to the Qiskit Runtime service.
+Primitives are base level operations that serve as building blocks for many quantum algorithms and applications. The `base primitive interfaces <https://qiskit.org/documentation/apidoc/primitives.html>`__ are defined in Qiskit Terra, and many Qiskit algorithms use the primitives natively. This abstraction allows you to write the same code, using Qiskit algorithms or otherwise, that can run on different quantum hardware or simulators without having to explicitly manage some of the finer details. See `Introduction to primitives <primitives.html>`__ for further details.
```

Comment:
If we are going to introduce the term "base primitive" here, then we should follow that up by saying the primitves offered by the `qiskit_ibm_runtime` add additional options specific to IBM's service.